### ISSUE TITLE
Scanning reports duplicates when files are not actually duplicates

### ISSUE THREAD

**Issue by rlitman** [Rello/audioplayer#369]
It looks like duplicates are detected by size, but when I look at a checksum of the reported duplicates, they prove to be actually different.

**Comment by Rello** [comment]
Hello @rlitman,

actually, its checking the ID3 information if the very same file is already there. The list of checked fields is here
https://github.com/Rello/audioplayer/blob/master/lib/Controller/DbController.php#L365-L381

so the file actually is the same content wise

**Comment by Rello** [comment]
Hello,
could you verify the mentioned double files?
if you are using the `occ` command, you get more detailed output

**Comment by rlitman** [comment]
Sorry for the delay.  I just checked from the command line, and all the duplicates I see from the occ command are real duplicates.  I think that maybe I was just confused by what I was looking at when I did the scan in my browser.

I did get this odd error from occ though:
Attention: Only slow indexing due to server config. See Audio Player wiki on GitHub for details.
I find  it odd, because I do not use external or SMB storage.  I actually have the files_external app disabled.

Anyway, thanks for following up, ans sorry for the confusion.
